### PR TITLE
Add dynamic pricing update

### PR DESCRIPTION
**Pull request by sousinha1997**
This code is to add dynamic pricing calculation for gcp and azure prices. Commented the code in main block.

### CODE REVIEW

File: quisby/pricing/cloud_pricing.py [hunk 1]
```diff
@@ -62,6 +65,36 @@ def get_azure_pricing(system_name, region="US Gov"):
...
+        return None
+    prefix=""
+    gcp_price_list = google_ext_prices["gcp_price_list"]
+    machine_fam = instance_name.split("-")[0].upper()
+    if machine_fam in ("N2","N2D", "T2D", "T2A", "C2", "C2D", "M1", "M2"):
+        prefix = "CP-COMPUTEENGINE-" + machine_fam + "-PREDEFINED-VM-CORE".strip()
+    elif machine_fam in ("N1", "E2"):
+        prefix = 'CP-COMPUTEENGINE-VMIMAGE-' + instance_name.upper().strip()
+    else:
+        logging.error("This machine prices not available")
```
Comment (frival) [inline quisby/pricing/cloud_pricing.py]: Micro-nit: "price is not"

File: quisby/pricing/cloud_pricing.py [hunk 2]
```diff
@@ -138,13 +174,17 @@ def get_cloud_cpu_count(instance_name, region, cloud_type):
...
     elif cloud_type == "local":
         return 1
 
 
 
 if __name__ == "__main__":
     region = "US East 2"
-    print(get_azure_pricing("Standard_D32s_v3", region))
+    #print(get_azure_pricing("Standard_D32s_v3", region))
     # print(get_aws_cpucount("i3en.24xlarge", "US East (N. Virginia)"))
```
Comment (frival) [inline quisby/pricing/cloud_pricing.py]: Should probably delete both of these since they're just debugging aids?

**Review by frival — APPROVED** [review]
Just two minor nits, your choice to clean them up first or do it in a follow-on.

File: quisby/pricing/cloud_pricing.py (hunk 1 above)
Comment (sousinha1997) [reply quisby/pricing/cloud_pricing.py]: Done

File: quisby/pricing/cloud_pricing.py (hunk 2 above)
Comment (sousinha1997) [reply quisby/pricing/cloud_pricing.py]: Yep

**Pull request merged**